frontend/webadmin/modules/uicommonweb/src/main/java/org/ovirt/engine/ui/uicommonweb/models/vms/UnitVmModel.java

```diff
@@ -2606,6 +2606,10 @@ private Set<DisplayType> getSupportedDisplayTypes(int osId, BiosType biosType) {
     }
 
     public void initDisplayModels(Set<DisplayType> displayTypes) {
+        if (displayTypes.remove(DisplayType.qxl)) {
+            // switching QXL to the least priority
+            displayTypes.add(DisplayType.qxl);
+        }
```

Comment:
`displayTypes` is an HashSet I presume and so the iteration order is not guaranteed
you probably want to have a data structure that maintains the order and to prefer types that are not Deprecated rather than having QXL hard coded - but I really don't think it is worth the effort..